js/script.js

```diff
@@ -1,15 +1,215 @@
-// Javascript Code.
-$(document).ready(function () {
-  // $.get("_posts/2014-11-09.json", function(data){
-  //
-  //   var posts = JSON.parse(data);
-  //   $(".posts").html(posts['posts'][0]['title']);
-  //
-  // });
-
-  $(this).load("_posts/posts.json", function(data){
-    var posts = JSON.parse(data);
-    $(".posts").html(posts['posts'][0]['title']);
-    //
-  });
-});
+// Javascript Code.

```

Comment:
You have two scripts, here in github , uploading you should only need because when your project qualify there is some confusion because it is not known which of the two scripts is used or you are using.